apps/win32/package.json

```diff
@@ -43,7 +45,27 @@
     "react": "16.11.0",
     "react-native": "0.62.2",
     "react-native-svg-transformer": "^0.14.3",
-    "react-test-renderer": "~16.11.0"
+    "react-test-renderer": "~16.11.0",
+    "react-native-windows": "0.0.0-master.52",
+    "appium": "1.17.1",
```

Comment:
this seems like a really old version?

Reply:
I just copied from other parts of our repo. This is what we had. Not sure why the older versions

Reply:
- Why does apps/win32 depend on react-native-windows at all?
- I don't see this version elsewhere on the repo,  we should use the same version in this PR and look at upgrading separately

Reply:
It doesn't, I removed it. As for the fluent-tester/package.json, I reverted it back to the version used elsewhere in the repo.